geoengine/workflow.py

```diff
@@ -189,10 +189,10 @@ def geo_json_with_time_to_geopandas(geo_json):
 
         return geo_json_with_time_to_geopandas(data)
 
-    def wms_get_map_as_image(self, bbox: QueryRectangle, colorizer_min_max: Tuple[float, float] = None) -> Image:
+    def wms_get_map_as_image(self, bbox: QueryRectangle, colorizer: str = None) -> Image:
```

Comment:
Also here, take a Colorizer.
Moreover, you should not allow None for querying WMS because there is no good default.